pkg/apis/extensions/install/install.go

```diff
@@ -43,11 +43,9 @@ var availableVersions = []unversioned.GroupVersion{v1beta1.SchemeGroupVersion}
 
 func init() {
```

Comment:
It is still valid. @deads2k has another PR related to this, but IIRC that one also relies on the `availableVersions` to be in the correct order.